resources/js/Components/CustomFieldDisplay.tsx

```
@@ -470,6 +473,55 @@ export default function CustomFieldDisplay({
             case "password":
                 return <span className="text-gray-500">••••••••</span>;
 
+            case "currency":
+                let currencyData: { amount: string | number | null; currency: string } | null = null;
+                
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)